src/main/java/com/fasterxml/jackson/databind/util/EnumValues.java

```diff
@@ -218,7 +256,7 @@ public EnumMap<?,SerializableString> internalMap() {
             for (Enum<?> en : _values) {
                 map.put(en, _textual[en.ordinal()]);
             }
-            result = new EnumMap(map);
+            _asMap = result = new EnumMap(map);
```

Comment:
Technically unrelated to this PR, but a performance improvement if `internalMap` is repeatedly called outside this module